CHANGELOG.md

```diff
@@ -19,6 +20,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   to provide the previously existing functionality.
 * The only required change for most downstream crates to adapt to the above is to
   make sure the `SerDe` trait is in scope (`use miniconf::SerDe`).
+* Paths now start with the path separator (unless they are empty).
```

Comment:
I think we need to note a change in miniconf_dervie as well. We'll need to republish both crates when we eventually release.

Reply:
`miniconf_derive` doesn't have a changelog or a readme. I think it's usually done like that. derive crates are included, described and tracked in the main user-facing crate.

Reply:
I just want something to remind me that we can't relese until the derive crate is out, but I _think_ that will already happen because we changed the API of the trait